### PR TITLE
Pharma - Add TXA autoinjector and fix coagulation, kinda

### PR DESCRIPTION
**Pull request by Cplhardcore**
**When merged this pull request will:**
- Adds a TXA autoinjector
- Applies a bandage to coag so all wound wont get bandaged all at once anymore (It works, but not well?) 

Todo, 
- [ ] Add a proper icon to the TXA autoinjector
- [x] Figure out why coag isnt using the values that is set for it in CBA settings

### IMPORTANT

- [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

File: addons/pharma/functions/fnc_handleRespawn.sqf [hunk 1]
```diff
@@ -111,25 +111,27 @@ if (GVAR(coagulation)) then {
...
-
-        if (_count == 0) exitWith {
+        private _TXAcount = [_unit, "TXA",] call ACEFUNC(medical_status,getMedicationCount);
+        private _TXAAutocount = [_unit, "TXAAuto",] call ACEFUNC(medical_status,getMedicationCount);
+        if (_TXAcount == 0 && _TXAAutocount == 0) exitWith {
             {
                 _x params ["", "_bodyPart", "_amount", "_bleeding"];
 
                 if (_amount * _bleeding > 0) exitWith {
+                    sleep ((GVAR(coagDelay)) + random(GVAR(coagRdmDelay)));
```
Comment (MiszczuZPolski) [inline addons/pharma/functions/fnc_handleRespawn.sqf]: https://ace3.acemod.org/wiki/development/coding-guidelines

**Comment by Cplhardcore** [comment]
Coag is fixed and working

File: addons/pharma/stringtable.xml [hunk 2]
```diff
@@ -3242,5 +3254,17 @@
         <Key ID="STR_KAT_Pharma_SETTING_EACA_Trauma_Desc">
             <English>Allows Trauma to be cleared by EACA in addition to stitching</English>
         </Key>
+        <Key ID="STR_KAT_Pharma_SETTING_Coagulation_Delay">
+            <English>Coagulation Delay</English>
+        </Key>
+        <Key ID="STR_KAT_Pharma_SETTING_Coagulation_Delay_DESC">
+            <English>How long it takes for coagulation to bandage a wound</English>
```
Comment (LinkIsGrim) [inline addons/pharma/stringtable.xml]:
```suggestion
            <English>Minimum time for coagulation to close a wound</English>
```

File: addons/pharma/stringtable.xml [hunk 3]
```diff
@@ -3242,5 +3254,17 @@
...
+        </Key>
+        <Key ID="STR_KAT_Pharma_SETTING_Coagulation_Random_Delay">
+            <English>Coagulation Delay Extension</English>
+        </Key>
+        <Key ID="STR_KAT_Pharma_SETTING_Coagulation_Random_Delay_DESC">
+            <English>How long it additionally takes for coagulation to bandage a wound, random number between 0 and this setting</English>
```
Comment (LinkIsGrim) [inline addons/pharma/stringtable.xml]:
```suggestion
            <English>Additional time coagulation can take to close a wound, random number between 0 and this setting</English>
```

**Pull request closed without merging**